api/routes/subcriterias/get_sub_criteria.py

```diff
@@ -74,11 +74,22 @@ def get_themes_fields(
 
 
 def get_all_uploaded_document_field_answers(
-    fund_id: str, round_id: str
+    fund_id: str,
+    round_id: str,
+    language: str,
 ) -> list[dict]:
     sub_criterias = get_all_subcriteria(fund_id, round_id)
     filtered_answers = [
-        {**answer, "question": f"{theme['name']}, {answer['question']}"}
+        {
```

Comment:
Is this the only place we pull back file uploads? I think on assessment frontend we may need to update some things to make sure these changes don’t break, from a quick look this endpoint only seems to be used for the “all uploaded documents” feature - what about regular sub criteria’s, did they work with this change?